### ISSUE TITLE
Project Admin Section

### ISSUE THREAD

**Issue by danmcp** [openshift/openshift-docs#2620]
It seems like we could use a project admin section in the docs.  I assume the project admin has to pick out their special logic from within the developer guide.   And it isn't always a distinct role.  But even if a developer is the project admin, it would be nice to have a place to look when trying to do project admin type things.  And for the editor that doesn't have access to the project admin logic, it would be that much less cruft they would see that they weren't allowed to run.

**Comment by mburke5678** [comment]
@danmcp If we were to create a project admin section, a call that is not mine, what would expect to see there? I did a quick scan of the table of contents and found these topics:

Architecture/Core Concepts/Projects
https://docs.openshift.com/container-platform/3.6/architecture/core_concepts/projects_and_users.html#projects

Cluster Admin/Managing Projects
https://docs.openshift.com/container-platform/3.6/admin_guide/managing_projects.html

Developer Guide/Projects
https://docs.openshift.com/container-platform/3.6/dev_guide/projects.html

Dev Guide/Compute Resources/Project Limits (link to another section) 
https://docs.openshift.com/container-platform/3.6/dev_guide/compute_resources.html#project-resource-limits

**Comment by danmcp** [comment]
@mburke5678 

I think I have changed my mind about this request since I opened it.  The problem still exists, but I think we should solve it a different way.  I am now of the opinion that the sections in the docs are fairly harmful.  The first thing I would want to do is get rid of the architecture section.  Then maybe consider getting rid of the dev vs admin sections.  The reason I am of this opinion is I have watched a lot of people try and use our docs over the past 6 months and seen them struggle.  The way people use the docs is to search for what they want and go to the most relevant page from the search results.  The problem is that they aren't making this selection based on being an architect, dev, or admin.  And so once they get to the page for the topic, if the relevant data isn't there, they will get frustrated.  I think a better alternative to the sections would be to instead just have everything split up by topics.  Once in a topic, it we should just clearly note when content is relevant to only a specific class of user.

Said another way, the sections only seem relevant to someone who is reading all the docs.  That's going to be a rare event.  And even if they are reading all the docs, it's doubtful they only care about 1 persona.
